projects/vdk-plugins/vdk-jupyter/build-job-tutorial-temp/tutorial.md

```diff
@@ -0,0 +1,269 @@
+# Learn How to Create and Build a Data Job through the Jupyter UI
+Time Commitment: About 25-30 minutes.
+
+## Prerequisites
+
+Before you start with the conversion, please ensure you have the following:
+
+<details>
+  <summary><b>Installed Software and Services</b></summary>
+
+- **VDK:** Ensure that the Versatile Data Kit is installed and properly configured.
+- **[Control Service](https://github.com/vmware/versatile-data-kit/wiki/Interfaces#control-service:~:text=Parameterized%20SQL-,Control%20Service,-Job%20Lifecycle%20API):** Essential for orchestrating the execution of Data Jobs.
+- **Jupyter instance:** Needed to access Jupyter Notebooks.
+- **[vdk-jupyterlab-extension](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/README.md):** This extension integrates VDK with JupyterLab.
+- **[vdk-notebook](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-notebook/README.md):** Allows the execution of VDK jobs with notebooks.
+- **[vdk-ipython](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-ipython/README.md):** Loads VDK functionalities into IPython environments, allowing enhanced interaction.
+
+</details>
+
+Make sure to have all prerequisites in place to avoid any disruptions during the data job creation and build process.
+
+## Data Job Creation Guide
```

Comment:
This should be separate tutorial , no ? 
 
And then included in pre-requisites.

Reply:
@antoniivanov i agree, it is best to be in another tutorial since this one ended up being pretty long. I will move it